test/common/wpt.js

```diff
@@ -459,6 +459,12 @@ class WPTRunner {
         );
         this.inProgress.delete(testFileName);
       });
+
+      await new Promise((resolve) => {
```

Comment:
If you want to `reject` when the `worker` emits an `error` event, you could also use
[`await events.once(worker, 'exit')`](https://nodejs.org/api/events.html#eventsonceemitter-name-options).

That being said, it looks like the caller never awaits the returned `Promise` (e.g., in `test-webcrypto.js`). What is the point in creating the `Promise` here if the caller does not await it?

Reply:
The point is to process the queue one by one. This has helped stabilize CI, especially the keygen tests that easily bogged the hosts down minutes at a time.

Reply:
> If you want to `reject` when the `worker` emits an `error` event, you could also use [`await events.once(worker, 'exit')`](https://nodejs.org/api/events.html#eventsonceemitter-name-options).

Rejecting would skip running the rest of the queue. So maybe this?

```js
await Promise.allSettled([events.once(worker, 'exit')])
```